DOCS/glauth-utils.md

```diff
@@ -24,10 +24,9 @@ project? [Get in touch with the team!](https://matrix.to/#/!nRbdoDYxdQndEfzlJi:u
 
 | Level | Path           | Navlink                                                       |
 |-------|----------------|---------------------------------------------------------------|
-| 1     | tutorial       | [Tutorial](/t/13949)                                 |
-| 1     | how-to         | [How-to guides](/t/<discourse-ID>)                            |
-| 2     | configure      | [Configure](/t/<discourse-ID>)                                |
-| 1     | reference      | [Reference](/t/<discourse-ID>)                                |
+| 1     | tutorial       | [Tutorial](/t/13949)                                          |
+| 1     | how-to         | How-to guides                                                 |
+| 1     | reference      | Reference                                                     |
```

Comment:
Are these placeholders?

Reply:
Yep, leave some placeholders for future complements.